docs/source/slurm/create-env.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
I don't mind having these files short-term in here, but would it make sense in the long run to split them into a separate repo maybe? It seems like they're very specific, but maybe I'm overlooking things.